include/deal.II/physics/elasticity/standard_tensors.h

```diff
@@ -0,0 +1,360 @@
+       *  := \mathcal{P} : \{ \bullet \} = \mathcal{P}^{T} : \{ \bullet \}
+       * @f]
+       * and, therefore,
+       * @f[
+       * \textrm{trace}(\textrm{dev} \{ \bullet \}) = 0 \, .
+       * @f]
+       * This definition aligns with the fourth-order SymmetricTensor $\mathcal{P}$
+       * mentioned in the class description and that returned by deviator_tensor().
+       */
+      static const SymmetricTensor<4, dim> dev_P;
+
+      /**
+       * Returns the fourth-order referential deviatoric tensor, as constructed from
+       * the deformation gradient tensor @p F.
+       * Also known as the deviatoric operator, this tensor projects a second-order
+       * SymmetricTensor onto a deviatoric space (for which the hydrostatic
+       * component is removed).
+       *
+       * This isochoric projection tensor is defined as
+       * @f[
+       *   \hat{\mathcal{P}}
+       *     := \frac{\partial \bar{\mathbf{C}}}{\partial \mathbf{C}}
+       * @f]
+       * with
+       * @f[
+       *  \bar{\mathbf{C}} := J^{-2/\textrm{dim}} \mathbf{C}
+       *    \qquad \text{,} \qquad
+       *  \mathbf{C} = \mathbf{F}^{T}\cdot\mathbf{F}
+       *    \qquad \text{and} \qquad
+       *  J = \textrm{det}\mathbf{F}
+       * @f]
+       * such that, for any second-order (referential) symmetric tensor,
+       * the following holds:
+       * @f[
+       *  \textrm{Dev} \{ \bullet \} = J^{-2/\textrm{dim}} \left[ \{ \bullet \} -
+       *  \frac{1}{\textrm{dim}}\left[\mathbf{C} : \{ \bullet \}\right] \mathbf{C}^{-1} \right]
+       *  := \{ \bullet \} : \hat{\mathcal{P}}  \, .
+       * @f]
+       * It can therefore be readily shown that
+       * @f[
+       *  \textrm{trace}(\textrm{Dev} \{ \bullet \}) = 0 \, .
+       * @f]
+       *
+       * @note It may be observed that we have defined the tensor as the
+       * transpose of that adopted by Wriggers (2008). We have done this so that
+       * it may be strictly applied through the chain rule to achieve the
+       * definition of the second Piola-Kirchhoff stress, i.e.
+       * @f[
+       *   \mathbf{S}
+       *     = 2\frac{\partial \psi \left( \bar{\mathbf{C}} \right)}{\partial \mathbf{C}}
+       *     = 2\frac{\partial \psi \left( \bar{\mathbf{C}} \right)}{\partial \bar{\mathbf{C}}}
+       *     : \frac{\partial \bar{\mathbf{C}}}{\partial \mathbf{C}}
+       *     = \bar{\mathbf{S}} : \hat{\mathcal{P}}
+       *     \equiv \hat{\mathcal{P}}^{T} : \bar{\mathbf{S}} \, .
+       * @f]
+       *
+       * @note Comparing the definition of this tensor in Holzapfel (2001) to that
+       * adopted here, the inclusion of the extra factor $J^{-2/\textrm{dim}}$ does not,
+       * at the outset, seem to be a reasonable choice.
+       * However, in the author's view it makes direct implementation of the
+       * expressions for isochoric (referential) stress contributions and their
+       * linearisation simpler in practise.
```

Comment:
Interestingly, as far as mother tongue language is concerned, English only ranks about 5 out of 11 for our national languages. So 🇿🇦 is definitely not representative in this case :-)